app/views/posts/show.html.erb

```diff
@@ -1 +1,42 @@
-<h1>Here is a particular article of index 1 for a user</h1>
\ No newline at end of file
+<section class="container">
+  <div class="post-info">
+    <div class="post-title-box">
+      <p class="post-title">Post : <%= @post.title %> by <%= @post.author.name %></p>
+     
+    </div>
+    <p><%= @post.text %></p>
+     <p class="comment-text">Comments: <%= @post.comments_counter || 0 %>, Likes: <%= @post.likes_counter || 0 %></p>
+
+     <div class = 'd-flex'>
+
+     <% if can? :destroy , @post %>
+     <%= button_to 'Delete Post', user_post_path(@user, @post), method: :delete, class: "btn btn-danger mx-3" %>
+     <% end %>
+
+     <%= render "likes/add_like"%>
+     </div>
+
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure the app/views/posts/show.html.erb view page math its given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png) as required for the project. I suggest you check each page design and match it to the expected [wireframe as in the project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)